### ISSUE TITLE
Notification is not working with t1.2.0 release

### ISSUE THREAD

**Issue by patrickelectric** [bluerobotics/ping-viewer#511]

**Comment by patrickelectric** [comment]
log:
```
...
[09:51:49:778] ping.networktool[Debug]: Testing version: "t0.9.0"
[09:51:49:778] ping.networktool[Debug]: Possible new version.
[09:51:49:778] ping.networktool[Debug]: comparing version: "t0.9.0" # 589824
[09:51:49:778] ping.networktool[Debug]: Testing version: "t0.8.5"
[09:51:49:778] ping.networktool[Debug]: Possible new version.
[09:51:49:778] ping.networktool[Debug]: comparing version: "t0.8.5" # 525568
[09:51:49:778] ping.networktool[Debug]: Testing version: "t0.8.4"
[09:51:49:778] ping.networktool[Debug]: Possible new version.
[09:51:49:779] ping.networktool[Debug]: comparing version: "t0.8.4" # 525312
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.8.2"
[09:51:49:779] ping.networktool[Debug]: Possible new version.
[09:51:49:779] ping.networktool[Debug]: comparing version: "t0.8.2" # 524800
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.8.1"
[09:51:49:779] ping.networktool[Debug]: Possible new version.
[09:51:49:779] ping.networktool[Debug]: comparing version: "t0.8.1" # 524544
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.8.0"
[09:51:49:779] ping.networktool[Debug]: Possible new version.
[09:51:49:779] ping.networktool[Debug]: comparing version: "t0.8.0" # 524288
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.8"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.72"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "continuous-csv"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "csv"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.71"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.7"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.61"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.6"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.5"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "continuous-Download_Example_Log"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.4"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.3"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:779] ping.networktool[Debug]: Testing version: "t0.2"
[09:51:49:779] ping.networktool[Debug]: Not a release. QRegularExpressionMatch(Valid, no match)
[09:51:49:785] ping.notificationmanager[Debug]: New Notification: "New version: t1.2.0 - <strong>Test Release (t1.2.0)</strong><br> <strong>Corrections and features around v1.1.1</strong><br> Changes since last version:<br>  - New crash report feature with DrKonqi<br>  - Fix restart application button<br>  - Update icons for HD/4K displays<br>  - Update user message when connection is closed<br>  - Code is now compatible with mycrosoft compiler (msvc) <br><a href=\"https://github.com/bluerobotics/ping-viewer/releases/download/t1.2.0/pingviewer_release.zip\">DOWNLOAD IT HERE!</a>"
[09:51:53:815] ping.protocol.protocoldetector[Debug]: Waiting port to open..  0 "COM18"
[09:51:53:815] ping.protocol.protocoldetector[Debug]: Probing Serial "COM18" 115200
[09:51:53:815] ping.protocol.protocoldetector[Debug]: Port is open
[09:52:24:394] ping.protocol.protocoldetector[Debug]: Waiting port to open..  0 "COM21"
[09:52:24:445] ping.protocol.protocoldetector[Debug]: Waiting port to open..  1 "COM21"
[09:52:24:495] ping.protocol.protocoldetector[Debug]: Waiting port to open..  2 "COM21"
[09:52:24:495] ping.protocol.protocoldetector[Debug]: Probing Serial "COM21" 9600
[09:52:24:512] ping.protocol.protocoldetector[Debug]: Port is open
[09:52:25:240] ping.protocol.protocoldetector[Debug]: Waiting port to open..  0 "COM21"
[09:52:25:290] ping.protocol.protocoldetector[Debug]: Waiting port to open..  1 "COM21"
[09:52:25:341] ping.protocol.protocoldetector[Debug]: Waiting port to open..  2 "COM21"
[09:52:25:341] ping.protocol.protocoldetector[Debug]: Probing Serial "COM21" 115200
[09:52:25:358] ping.protocol.protocoldetector[Debug]: Port is open
[09:52:25:490] ping.protocol.protocoldetector[Debug]: Ping detected on: "LinkConfiguration{Name: Detector serial link, LinkType: 2, Arguments: (COM21:115200)}"
[09:52:25:498] ping.protocol.sensor[Debug]: Connecting to "LinkConfiguration{Name: Detector serial link, LinkType: 2, Arguments: (COM21:115200)}"
[09:52:25:501] ping.protocol.seriallink[Debug]: "LinkConfiguration{Name: Detector serial link, LinkType: 2, Arguments: (COM21:115200)}"
[09:52:25:522] default[Warning]: ConnectionMenu.qml(44) qrc:/ConnectionMenu.qml:44: TypeError: Cannot assign to read-only property "currentText"
[09:52:25:522] ping.protocol.ping[Debug]: Start pre configuration task and requests.
[09:52:25:522] ping.settingsmanager[Debug]: "Save lastLinkConfiguration with:" "LinkConfiguration{Name: Detector serial link, LinkType: 2, Arguments: (COM21:115200)}"
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1215
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1205
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1300
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1200
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1201
[09:52:25:522] ping.protocol.ping[Debug]: Requesting: 1203
[09:52:25:522] ping.filemanager[Debug]: Creating file name: "C:/Users/Brian/Documents/PingViewer/Sensor_Log/20190603-095225522.bin"
[09:52:25:526] ping.protocol.filelink[Debug]: "LinkConfiguration{Name: , LinkType: 1, Arguments: (C:/Users/Brian/Documents/PingViewer/Sensor_Log/20190603-095225522.bin:w)}"
[09:52:25:532] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:533] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:533] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:533] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:534] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:534] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:534] ping.settingsmanager[Debug]: "Load enabledCategories with:" 1048574
[09:52:25:535] ping.protocol.ping[Debug]: Handling Message: 1215
[09:52:25:535] ping.protocol.ping[Debug]: Requesting: 1215
[09:52:25:536] ping.protocol.ping[Debug]: 
PingConfiguration {
	1_pingInterval: 20
	1_speedOfSound: 1500000
	2_automaticMode: 0
	3_gainIndex: 0
	3_lengthDistance: 5000
	3_startDistance: 0
}
[09:52:25:536] ping.protocol.ping[Debug]: Handling Message: 1205
[09:52:25:537] ping.protocol.ping[Debug]: Requesting: 1205
[09:52:25:538] ping.protocol.ping[Debug]: Handling Message: 1200
[09:52:25:540] ping.protocol.filelink[Debug]: File will be opened.
[09:52:25:612] ping.protocol.ping[Debug]: Handling Message: 1201
[09:52:25:614] ping.protocol.ping[Debug]: Handling Message: 1203
[09:52:25:616] ping.protocol.ping[Debug]: Handling Message: 1215
[09:52:25:617] ping.protocol.ping[Debug]: Handling Message: 1205
[09:52:25:800] ping.protocol.ping[Debug]: "Ping_V3.26_115kb.hex"
[09:52:25:800] ping.protocol.ping[Debug]: "Ping_V3.26_9600b.hex"
[09:52:25:846] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:25:997] ping.protocol.protocoldetector[Debug]: Scan finished.
[09:52:26:041] ping.protocol.ping[Debug]: "Device configuration does not match. Waiting for (20), got (0) for 1_pingInterval"
[09:52:26:041] ping.protocol.ping[Debug]: Requesting: 1206
[09:52:26:041] ping.protocol.ping[Debug]: "Device configuration does not match. Waiting for (0), got (1) for 2_automaticMode"
[09:52:26:041] ping.protocol.ping[Debug]: Requesting: 1205
[09:52:26:041] ping.protocol.ping[Debug]: "Device configuration does not match. Waiting for (0), got (6) for 3_gainIndex"
[09:52:26:041] ping.protocol.ping[Debug]: Requesting: 1207
[09:52:26:041] ping.protocol.ping[Debug]: "Device configuration does not match. Waiting for (5000), got (10922) for 3_lengthDistance"
[09:52:26:041] ping.protocol.ping[Debug]: Requesting: 1204
[09:52:26:055] ping.protocol.ping[Debug]: Handling Message: 1206
[09:52:26:057] ping.protocol.ping[Debug]: Handling Message: 1205
[09:52:26:058] ping.protocol.ping[Debug]: Requesting: 1205
[09:52:26:070] ping.protocol.ping[Debug]: Handling Message: 1207
[09:52:26:072] ping.protocol.ping[Debug]: Handling Message: 1204
[09:52:26:074] ping.protocol.ping[Debug]: Handling Message: 1205
[09:52:26:518] ping.protocol.ping[Debug]: Requesting: 1214
[09:52:26:518] ping.protocol.ping[Debug]: Requesting: 1213
[09:52:26:519] ping.protocol.ping[Debug]: Requesting: 1202
[09:52:26:519] ping.protocol.ping[Debug]: Requesting: 1205
[09:52:26:534] ping.protocol.ping[Debug]: Handling Message: 1214
[09:52:26:536] ping.protocol.ping[Debug]: Handling Message: 1213
[09:52:26:538] ping.protocol.ping[Debug]: Handling Message: 1202
[09:52:26:540] ping.protocol.ping[Debug]: Handling Message: 1205
[09:52:26:542] ping.protocol.ping[Debug]: Last configuration done, timer will stop now.
[09:52:27:043] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:053] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:055] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:058] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:060] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:061] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:063] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:065] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:068] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:070] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:072] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:074] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:076] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:078] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:080] ping.protocol.ping[Debug]: Handling Message: 1300
[09:52:27:082] ping.protocol.ping[Debug]: Handling Message: 1300
```

**Comment by patrickelectric** [comment]
Fixed in #672
